scribble-doc/scribblings/scribble/core.scrbl

```diff
@@ -1421,7 +1444,32 @@ number for each subsection. If @racket[numberer-step] produces a plain
 string for the rendered number, then it is not added as a prefix to
 subsection numbers. See also @racket[collected-info].
 
-@history[#:added "6.4"]}
+@history[#:added "1.1"]}
+
+
+@defstruct[link-render-style ([mode (or/c 'default number)])]{
```

Comment:
typo: `number` => `'number`

About naming: I'm thinking `'default` should be `'all`, so it's phrased as an answer to the question "what parts of the `linke-element` are clickable?"